.github/workflows/release.yml

```diff
@@ -72,6 +72,7 @@ jobs:
     permissions:
       id-token: write
       attestations: write
+      contents: write
```

Comment:
The failure caused by the lack of content write permission can be seen here: https://github.com/pex-tool/pex/actions/runs/9684301905/job/26722997959